bluetooth/src/androidLibMain/kotlin/device/DeviceConnectionManager.kt

```diff
@@ -197,6 +196,12 @@ internal actual class DeviceConnectionManager(
         }
     }
 
+    override fun unpair() {
```

Comment:
Basically, this function is not unpair but starts unpairing.  I guess it should be renamed to `startUnpairing` or even better implement as a suspend function so it will wait until the device is actually unpaired

Reply:
In this particular case yes, but it's interface method override, other class can override it to unpair instantly who knows. But indeed it's better to introduce pairing state and make this method suspend in future.

https://github.com/splendo/kaluga/issues/519